src/include/cart/iv.h

```diff
@@ -343,6 +343,23 @@ typedef int (*crt_iv_on_put_cb_t)(crt_iv_namespace_t ivns,
 typedef bool (*crt_iv_keys_match_cb_t)(crt_iv_namespace_t ivns,
 				crt_iv_key_t *key1, crt_iv_key_t *key2);
 
+/**
+ * If provided, this callback will be called before the
+ * synchronization/notification is propagated (flowing down from root to leaf)
+ * to the child nodes, if any.
+ *
+ * \param[in] ivns		the local handle of the IV namespace
+ * \param[in] iv_key		key of the IV
+ * \param[in] iv_ver		version of the IV
+ * \param[in] iv_value		IV value to be refresh
```

Comment:
formatting issue it seems

Reply:
I've double checked in my Vim. The formatting is correct; the GitHub UI displays it incorrectly.